packages/generators/src/init-generator.ts

```diff
@@ -99,8 +99,15 @@ export default class InitGenerator extends CustomGenerator {
         }
     }
 
-    public installPlugins(): void {
-        const packager = this.utils.getPackageManager();
+    public async installPlugins(): Promise<void> {
+        const { packager } = await Question.List(
+            this,
+            "packager",
+            "Pick a package manager:",
+            this.utils.getAvailableInstallers(),
+            "npm",
```

Comment:
Maybe we can improve this, if develop has `yarn.lock` I think better to use `yarn` as default?